README.md

```diff
@@ -4,6 +4,8 @@
 
 Install dependencies & start:
 
+    nvm use 15
```

Comment:
Good one. You may also create an `.nvmrc` by running `echo 15 > .nvmrc && git add -f .nvmrc`

Reply:
Not ready for this, Netlify running on a different node version then 15.